src/Umbraco.Web/Search/SearchableTreeCollection.cs

```diff
@@ -19,15 +19,17 @@ public SearchableTreeCollection(IEnumerable<ISearchableTree> items, IApplication
 
         private Dictionary<string, SearchableApplicationTree> CreateDictionary(IApplicationTreeService treeService)
         {
-            var appTrees = treeService.GetAll().ToArray();
+            var appTrees = treeService.GetAll()
+                .OrderBy(x => x.SortOrder)
+                .ToArray();
             var dictionary = new Dictionary<string, SearchableApplicationTree>();
             var searchableTrees = this.ToArray();
-            foreach (var searchableTree in searchableTrees)
+            foreach (var appTree in appTrees)
             {
-                var found = appTrees.FirstOrDefault(x => x.Alias == searchableTree.TreeAlias);
+                var found = searchableTrees.FirstOrDefault(x => x.TreeAlias == appTree.Alias);
```

Comment:
Should we be case-sensitive (I know we were not, but...)?

Reply:
Yup, i can update